example/ipsec_crypto/odp_ipsec_cache.c

```diff
@@ -95,7 +95,7 @@ int create_ipsec_cache_entry(sa_db_entry_t *cipher_sa,
 		params.cipher_alg  = cipher_sa->alg.u.cipher;
 		params.cipher_key.data  = cipher_sa->key.data;
 		params.cipher_key.length  = cipher_sa->key.length;
-		params.cipher_iv.data = entry->state.iv;
+		params.cipher_iv.data = NULL;
 		params.cipher_iv.length = cipher_sa->iv_len;
```

Comment:
If `data` above is `NULL`, then this probably should be set to 0.

Reply:
I was getting ahead of myself related to the coming (proposed) deprecation of cipher_iv.data in the parameters. In the current version of the API the data cannot be omitted (even if it gets never used and even though linux-gen accepts a null pointer).

Fixed the problem in v2 by setting the IV data pointer point to a dummy array to make the code comply with the API spec even though the IV data in the session will not be used.